tests/js/app/testHistoryStore.js

```diff
@@ -172,20 +177,44 @@ describe('History Store', function () {
                 original_uri: 'http://endlessm.com',
             }),
         });
+        let share_dialog;
 
-        spyOn(Gtk, 'show_uri').and.callFake(function(a,uri,c) {
-            expect(uri).toMatch('.*' + network_uri_matcher + '.*');
-        });
+        if (network === HistoryStore.Network.FACEBOOK) {
+            let WebShareDialogConstructor = WebShareDialog.WebShareDialog;
+
+            spyOn(WebShareDialog, 'WebShareDialog').and.callFake(function(props) {
```

Comment:
Make me reconsider the decision of having a UI elem in HistoryStore heh, but this un-breaks our tests..